test/test_jit.py

```diff
@@ -6400,6 +6401,48 @@ def run_test(code):
                 code = funcs_template.format(func=func, scalar1=scalar1, scalar2=scalar2)
                 run_test(code)
 
+    def test_python_math(self):
+        funcs_two_argument_template = dedent('''
+        def func():
+            return {func}({scalar1}, {scalar2})
+        ''')
+        funcs_two_argument = ['math.pow']
+        funcs_one_argument_template = dedent('''
+        def func():
+            return {func}({scalar1})
+        ''')
+        funcs_one_argument = ['math.floor', 'math.ceil', 'math.log', 'math.log1p', 'math.log10', 'math.exp', 'math.sqrt',
+                              'math.pow', 'math.isnan', 'math.asinh', 'math.atanh', 'math.cosh', 'math.sinh', 'math.tanh']
+        scalars_float = ['-1000000.0', '-1000.0', '-2.0', '0.0', '1.0', '3.0', '1000.0', '10000000.0']
+        scalars_int = ['-1000000', '-1000', '-2', '0', '1', '3', '1000', '10000000']
```

Comment:
Test `NaN` and `inf` (both + and -) as well

Reply:
Will work on now

Reply:
Done will push soon

Reply:
This was fixed in the section that was removed due to the early request, let me know if I should add it back in or add it into the other set of tests